README.md

```diff
@@ -42,6 +42,12 @@ php artisan translatable:export es,bg,de
 
 The command with the "es,bg,de" parameter passed will create es.json, bg.json, de.json files with translatable strings or update the existing files in the `resources/lang` folder of your project.
 
+### Find untranslated lines
```

Comment:
@kohlerdominik Could you please explain why is this approach necessary and when? When there are text strings without any wrappers, such as `__()` or `<lang></lang>`?

Reply:
Hi

thanks for your answer. This is actually not for searching string in the code, it's rather for finding them in the language json files like:
![grafik](https://user-images.githubusercontent.com/18621527/50024851-f782be80-ffe3-11e8-803e-48d38fcc9aa5.png)

I updated my PR, hope it's more clear now.

Reply:
@kohlerdominik Ah, now I see what you meant. It's really handy. Thank you very much for this change!